src/form.js

```diff
@@ -624,6 +632,13 @@ define([
                 this.vellum.setUnsavedDuplicateNodeId(value);
                 return null;
             }
+            else if (property === 'defaultValue' && previous &&
```

Comment:
This PR seems to detect choice mugs (*Items*) by whether the mug has a `defaultValue` or not. I think it would be better to also check the class name (`mug.__className === "Item"`), since as far as I know there is nothing preventing other mug types from having a `defaultValue`.